pvlib/pvsystem.py

```diff
@@ -1122,6 +1127,8 @@ def retrieve_sam(name=None, path=None):
         elif name == 'sandiamod':
             csvdata = os.path.join(
                 data_path, 'sam-library-sandia-modules-2015-6-30.csv')
+        elif name == 'adrinverter':
+            csvdata = os.path.join(data_path, 'adr-library.csv')
```

Comment:
The parameters in this particular file were generated using the CEC efficiency measurements available to me as of 2013-10.  It might be good to add the date suffix.  I'm looking into doing an update.